File: src/test/java/app/pdfx/prefs/LocalDataDirTest.java
package app.pdfx.prefs;

import org.junit.jupiter.api.Test;

import java.io.FileNotFoundException;

import static org.junit.jupiter.api.Assertions.assertFalse;
import static org.junit.jupiter.api.Assertions.assertTrue;

public class LocalDataDirTest {

    @Test
    public void test_Unix_GetAppDirectory_DirectoryExists() throws FileNotFoundException {
        final var path = LocalDataDir.getAppPath("test");
        final var file = path.toFile();

        assertTrue(file.exists());
        assertTrue(file.delete());
        assertFalse(file.exists());
    }
}

File: src/test/kotlin/app/pdfx/prefs/LocalDataDirTest.kt
package app.pdfx.prefs

import org.junit.jupiter.api.Assertions.assertFalse
import org.junit.jupiter.api.Assertions.assertTrue
import org.junit.jupiter.api.Test

class LocalDataDirTest {

    @Test
    fun `should get app data directory`() {
        val path = LocalDataDir.getAppPath("test")
        val file = path.toFile()
        assertTrue(file.exists())
        assertTrue(file.delete())
        assertFalse(file.exists())
    }
}
